wp-content/plugins/hurumap/db.php

```diff
@@ -100,8 +100,20 @@ function migrate() {
     }
     if ($version < 4) {
         $wpdb->query("DROP TABLE IF EXISTS `{$wpdb->base_prefix}hurumap_charts`");
+        $wpdb->query("DROP TABLE IF EXISTS `{$wpdb->base_prefix}chart_sections`");
 
-        $sql = "CREATE TABLE `{$wpdb->base_prefix}hurumap_charts`(
+        $sql = "CREATE TABLE `{$wpdb->base_prefix}chart_sections`(
+            `id`         varchar(45) NOT NULL,
+            `name`       varchar(255) NOT NULL ,
+            `description` varchar(255) NOT NULL ,
+            `order`         int NOT NULL,
+            `published` tinyint NOT NULL DEFAULT 0,
+            `created_at` datetime NOT NULL DEFAULT NOW() ON UPDATE NOW(),
+            `updated_at` datetime NOT NULL DEFAULT NOW(),
```

Comment:
I think these are reversed, `created_at` should never be updated _after_ the row is created. `updated_at` is the one we want to update on every row update.